sdo/grant_sdo_context.jsonld

```diff
@@ -10,6 +10,34 @@
     "name": {
       "@id": "sdo:name",
       "@type": "sdo:Text"
+    },
+    "alternateIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "relatedIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "funds": {
+      "@id": "sdo:fundedItem",
+      "@type": "sdo:Thing"
+    },
+    "funders": {
+      "@id": "sdo:funder",
+      "@type": "sdo:Thing"
+    },
+    "awardees": {
+      "@id": "sdo:endorsee",
+      "@type": "sdo:Thing"
+    },
+    "dates": {
+      "@id": "sdo:temporalCoverage",
+      "@type": "sdo:Thing"
```

Comment:
Should be one of these data types DateTime, Text or URL. The `Text ` probably is the best option.